src/main/resources/static/templates/CSRFVulnerability/LEVEL_6/CSRFVulnerability.js

```diff
@@ -0,0 +1,41 @@
+function addingEventListenerToSendButton() {
+  document.getElementById("sendBtn").addEventListener("click", function () {
+    let url = getUrlForVulnerabilityLevel();
+    let dataPost = new FormData();
+    dataPost.append("id", document.getElementById("id").value);
+    dataPost.append("pass", document.getElementById("pass").value);
+    dataPost.append("newpass", "");
+    dataPost.append("rtkparam", "");
```

Comment:
No need to send the parameters which are not used. (you can configured json ignore on the java pojo class.)